CMakeLists.txt

```diff
@@ -76,6 +76,10 @@ else()
 			OUTPUT_STRIP_TRAILING_WHITESPACE)
 endif()
 
+if(WIN32)
```

Comment:
This check is supposed to stop people from trying to compile pmemfile on Windows just because they see if (WIN32) in the repository (like few lines below).